packages/kbn-generate-console-definitions/src/generate_query_params.test.ts

```diff
@@ -106,270 +63,4 @@ describe('generateQueryParams', () => {
       property2: '',
     });
   });
-
-  it('converts builtin types', () => {
```

Comment:
This code is moved to `./helpers/convert_url_properties.test.ts` because it is testing the functionality that is now extracted into `./helpers/convert_url_properties.ts`